llvm/lib/Target/AArch64/GISel/AArch64PostLegalizerCombiner.cpp

```diff
@@ -439,6 +441,19 @@ class AArch64PostLegalizerCombiner : public MachineFunctionPass {
 private:
   bool IsOptNone;
   AArch64PostLegalizerCombinerImplRuleConfig RuleConfig;
+
+
+  struct StoreInfo {
+    GStore *St;
+    GPtrAdd *Ptr;
+    int64_t Offset;
+    LLT StoredType;
```

Comment:
tiny optional nit: I'd prefer seeing default values for the first 3 fields - makes it easier to tell what's going on if we access some uninitialized data on accident.